save-common/src/jvmMain/kotlin/org/cqfn/save/entities/AgentStatus.kt

```diff
@@ -0,0 +1,15 @@
+package org.cqfn.save.entities
+
+import org.cqfn.save.agent.AgentState
+import java.time.LocalDate
```

Comment:
just a remark to @aktsay6: https://docs.liquibase.com/change-types/community/add-unique-constraint.html
this table should have unique constraints. Keep this in mind